lib/github.js

```diff
@@ -85,53 +86,89 @@ function aliasGet (msg, done) {
 }
 
 function queryGithub (msg, done) {
-  var cache = opts.cache
+  var cache  = opts.cache
 
   github.authenticate({
-    type: 'token',
+    type : 'token',
     token: opts.token
   })
 
-  var params = {
-    user: msg.user,
-    repo: msg.repo
-  }
-
-  github.repos.get(params, (err, repo) => {
-    if (err) return done(err)
-    if (!repo) return done()
+  async.parallel({
+    getRepository  : function (callbackFn) {
+      github.repos.get({
+        user: msg.user,
+        repo: msg.repo
+      }, callbackFn)
+    },
+
+    getReadme      : function (callbackFn) {
+      github.repos.getReadme({
+        user: msg.user,
+        repo: msg.repo
+      }, (err, readme_b64) => {
+        if (err) {
+          callbackFn(err, null)
+        } else {
+          if (readme_b64) {
+            github.misc.renderMarkdownRaw({
+              data: new Buffer(readme_b64.content, 'base64').toString("ascii")
+            }, (err, readme) => {
+              if (err && !readme) { // GitHub NodeJS API fails expecting a JSON response
+                callbackFn(err, null)
```

Comment:
GitHub API expects a JSON object. However, the service returns HTML code and the library fails to parse the response, hence err is always set. But it is checked for the case where there is an error and there is no content in the readme variable.